### ISSUE TITLE
Is preamp applied when downloading from AutoEQ?

### ISSUE THREAD

**Issue by bloatmode** [ThePBone/DDCToolbox#12]
I'm asking because after importing a parametric EQ from AutoEQ (using Project > Download from AutoEQ...) I see that all the Peaking filters in the project window correspond to the ones listed in my headphones' page in AutoEQ; however I don't see a Unity Gain filter, which should be necessary in order to apply the recommended preamp.

**Comment by timschneeb** [comment]
No, it is currently not applied. When I implemented this, AutoEq didn't include the preamp in the ParametricEQ.txt files yet, so I didn't bother parsing the accompanying readme file to get that value. 
Anyway, it is now included and can be parsed without problems, so I'll add this when I have time.
